lua/barbar/events.lua

```diff
@@ -213,12 +213,17 @@ function events.enable()
             group = augroup_render,
           })
 
-          create_autocmd(option.event or 'BufWinLeave', {
+          local close_events = { "BufWinLeave" }
+          if not vim.tbl_contains(close_events, option.event) then
```

Comment:
We can probably just do `close_events[1] ~= option.events` (since we know the structure of that particular table) right? `tbl_contains` iterates over the table to find a match which might not be necessary in this case

Reply:
That's true, thanks